src/redux/gas.ts

```diff
@@ -485,7 +485,11 @@ export const gasUpdateTxFee = (
       ? ethereumUtils.getEthPriceUnit()
       : ethereumUtils.getMaticPriceUnit();
 
-  if (isEmpty(gasFeeParamsBySpeed)) return;
+  if (
+    isEmpty(gasFeeParamsBySpeed) ||
+    (txNetwork === networkTypes.optimism && l1GasFeeOptimism === null)
+  )
+    return;
```

Comment:
at first we are loading the optimism fee and then from the send or transaction confirmation screens, displaying two different fees. So I added this to get the fees ready waiting for `l1GasFeeOptimism` to be defined for ex. from here https://github.com/rainbow-me/rainbow/blob/develop/src/screens/TransactionConfirmationScreen.js#L466